TumeKyouen/Network/RealtimeBattleRooms.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  RealtimeBattleRooms.swift
+//  TumeKyouen
+//
+//  Created by 石倉 昇 on 2016/11/16.
+//  Copyright © 2016年 noboru. All rights reserved.
+//
+
+import APIKit
+import Himotoki
+
+struct RealtimeBattleRoomRequest: TumeKyouenRequestType {
+    typealias Response = [RealtimeBattleRoom]
+
+    var method: HTTPMethod {
+        return .GET
+    }
+
+    var path: String {
+        return "/realtime/room"
+    }
+    func responseFromObject(object: AnyObject, URLResponse: NSHTTPURLResponse) throws -> Response {
+        return try decodeArray(object)
+    }
+}
+
+struct RealtimeBattleRoom: Decodable {
+    let id: Int
```

Comment:
ERROR: [Warning][variable_name] Variable Name
Variable name should be between 3 and 40 characters long: 'id'